_includes/components/header--extended.html

```diff
@@ -1,8 +1,6 @@
 {% if header %}
 <header class="usa-header usa-header-{{ header.type | default: 'extended' }}" role="banner">
```

Comment:
Same here, I think this could just be `usa-header-extended` instead of `{{ header.type | default: 'extended' }}`.